docs/pages/resources.rst

```diff
@@ -0,0 +1,131 @@
+.. _resources:
+
+Resources
+=========
+
+.. automodule:: runeq.resources
+
+For example usage patterns, see :ref:`quickstart`.
+
+API Clients
+-----------
+
+.. automodule:: runeq.resources.client
+
+.. autofunction:: runeq.resources.client.initialize
+.. autofunction:: runeq.resources.client.global_graph_client
+.. autofunction:: runeq.resources.client.global_stream_client
+
+.. autoclass:: runeq.resources.client.GraphClient
+   :special-members: __init__
+   :members:
+
+.. autoclass:: runeq.resources.client.StreamClient
+   :special-members: __init__
+   :members:
+
+Patient Metadata
+----------------
+
+.. automodule:: runeq.resources.patient
+
+Patients
+********
+
+.. autofunction:: get_patient
+.. autofunction:: get_all_patients
+
+.. autoclass:: Patient
+   :special-members: __init__
+   :members:
+   :inherited-members:
+.. autoclass:: PatientSet
+   :special-members: __init__
+   :members:
+   :inherited-members:
+
+
+Devices
+*******
+
+.. autofunction:: get_device
+.. autofunction:: get_all_devices
+
+.. autoclass:: Device
+   :special-members: __init__
+   :members:
+   :inherited-members:
+.. autoclass:: DeviceSet
+   :special-members: __init__
+   :members:
+   :inherited-members:
+
+Organization Metadata
+---------------------
```

Comment:
Is it possible to add a note on what an org is somewhere around here? Just something simple, like, "A research lab or clinical site is typically represented as an organization."

Reply:
yup! ditto, will fix in the follow up